src/java/org/apache/cassandra/config/YamlConfigurationLoader.java

```diff
@@ -157,7 +160,7 @@ public static <T> T fromMap(Map<String,Object> map, boolean shouldCheck, Class<T
         constructor.setPropertyUtils(propertiesChecker);
         Yaml yaml = new Yaml(constructor);
         Node node = yaml.represent(map);
-        constructor.setComposer(new Composer(null, null)
+        constructor.setComposer(new Composer(new ParserImpl(null), new Resolver(), new LoaderOptions())
```

Comment:
question:
Maybe it would be safe to add a comment about why it is safe to pass a `null` scanner to the parser? IIUC it is the case as the paper is never called to the overwritten `getSingleNode`, is that true?

Reply:
When I looked into that the release notes iirc mentioned they now switched to forcing you to specify all these that came as defaults in earlier versions. What I did was to mimic the behavior we had before.